app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, vstr: dict) -> str:
+        if "vaccine" not in vstr.keys() or vstr["vaccine"] == None:
```

Comment:
Use dct.get(key) python feature. Returns None if key doesn't exists.
```suggestion
        vaccine: dict = visitor.get("vaccine")
        if not vaccine:
            raise NotVaccinatedError("not vaccinated")```

Reply:
I should Note, that you NEVER compare a python value with None using `==`. You should do it `assert value is None`